doc/source/User_guide/settings.rst

```diff
@@ -0,0 +1,141 @@
+Settings
+========
+
+The Settings class is designed to handle the configurations of PyAEDT and AEDT.
+This includes behavior for logging, LSF scheduler, environment variable and general
+settings. Most of the default values used can be modified using a YAML configuration file.
+The path to this YAML file should be defined through the environment variable
+``PYAEDT_LOCAL_SETTINGS_PATH``. If the environment variable is set and a file exists,
+the default configuration settings are updated according to the content of the file.
+If the environment variable is not defined, a check if performed to see if a file named
+``"pyaedt_settings.yaml"`` exist in the user's ``APPDATA`` folder for Windows and
+``HOME`` folder for Linux. If such file exists, it is then used to update the default
+configuration.
+
+Bellow  is the content that can be updated through the YAML file. 
```

Comment:
```suggestion
Below  is the content that can be updated through the YAML file. 
```
I will let have a look at bellow verb definition. Most likely not what you meant 😃